TuT-103: Added Login Screen

arynshaikh opened the pull request:
- Modified activity_main.xml to give username and password field
- Added dimens.xml to give textsize to padding to linear layout

arynshaikh committed:
Tut-103: Added Login Screen

- Modified activity_main.xml to give username and password field
- Added dimens.xml to give textsize to padding to linear layout

arynshaikh commented:
![device-2018-09-30-172344](https://user-images.githubusercontent.com/38856465/46257354-c7904880-c4d5-11e8-87c3-4c0aa34d0fea.png)

nuclearsagar commented:
This is approved.

theyogeshrathod commented on app/src/main/res/values/dimens.xml:
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <dimen name="linear_padding">15dp</dimen>
+    <dimen name="text_size">20sp</dimen>
+    <dimen name="button_margin">40dp</dimen>
```
Rename `linear_padding -> login_layout_padding`
Rename `button_margin  -> login_button_margin`

arynshaikh replied:
Okay

theyogeshrathod replied:
Please make this change in the same branch:
Follow these steps:
- git checkout pr/arynshaikh/TuT-103-create-login-screen
- git pull
- Make changes in Android Studio
- git diff
- git add .
- git commit
- git push origin HEAD

theyogeshrathod commented:
Rename commit message title: 
`TuT-103: Added Login Screen` to `TUT-103: Added Login Screen`

arynshaikh committed:
TUT-103: Login Screen

- Modified the dimens.xml to change the name from
  1)linear_padding -> login_layout_padding
  2)button -> login_button_margin
